install/v0.6.0/kfserving.yaml

```diff
@@ -16642,23 +16642,23 @@ data:
         },
         "pmml": {
             "image": "kfserving/pmmlserver",
-            "defaultImageVersion": "latest",
+            "defaultImageVersion": "v0.6.0",
             "supportedFrameworks": [
               "pmml"
             ],
             "multiModelServer": false
         },
         "lightgbm": {
             "image": "kfserving/lgbserver",
-            "defaultImageVersion": "latest",
+            "defaultImageVersion": "v0.6.0",
             "supportedFrameworks": [
               "lightgbm"
             ],
             "multiModelServer": false
         },
         "paddle": {
             "image": "kfserving/paddleserver",
-            "defaultImageVersion": "latest",
+            "defaultImageVersion": "v0.6.0",
             "supportedFrameworks": [
               "paddle"
             ],
```

Comment:
Do we need these changes in `install/v0.6.0/kfserving.yaml`? They are not in the original PR